packages/react-scripts/scripts/utils/createJestConfig.js

```diff
@@ -37,7 +37,8 @@ module.exports = (resolve, rootDir, isEjecting) => {
       //   - ignore tests in dependencies
       // scripts
       //   - directory generated upon eject
-      '<rootDir>[/\\\\](build|\\.cache|docs|node_modules|scripts)[/\\\\]',
+      '<rootDir>/!(build|docs|node_modules|scripts)/**/__tests__/**/*.js?(x)',
```

Comment:
Since it is still in the master branch: Shouldn't that have been `.jsx?` instead of `.js?x`?